tools/pipelines-tasks/test/testhelpers.ts

```diff
@@ -58,6 +61,7 @@ export const expectedFilePath = (fileName: string): string =>
 export const TaskEntryPoints = {
     AppAttachTask: path.join(__dirname, '..', 'MsixAppAttach', 'index.js'),
     AppInstallerTask: path.join(__dirname, '..', 'AppInstallerFile', 'index.js'),
+    AVDAppAttachPublishTask: path.join(__dirname, '..', 'AVDAppAttachPublish', 'main.js'),
```

Comment:
This might be not relevant anymore, but for other TaskEntryPoints the file naming convention is `index.js`. For AVDAppAttachPublish, here `main.js` is used.

Reply:
Yes, but I have followed main.js as it will be clear where the trigger point of this task starts from.